try:
  print("try block1")
  try:
    print("try block2")
  except:
    print("exception2")
except:
  print("exception1")
